http_requests.py

```diff
@@ -75,40 +77,70 @@ def mount_default_adapter(
     return session
 
 
+def _create_logging_client():
+    '''Create an ELS Client for the default els in the active config set and return it
+
+    Returns `None` if the config set has no default ELS config. Raises an error if the active
+    config set cannot be determined (`CONCOURSE_CURRENT_CFG` env var must be set)
+    '''
+    config_set_name = util.check_env('CONCOURSE_CURRENT_CFG')
+    config_set = ctx().cfg_factory().cfg_set(config_set_name)
+    try:
+        elastic_cfg = config_set.elasticsearch()
+    except KeyError:
+        return None
+    return ccc.elasticsearch.from_cfg(elasticsearch_cfg=elastic_cfg)
+
+
+def _log_entry_consumer(
```

Comment:
naming nit: this is a function, not a class. I think functions ought rather be named with verbs as opposed to substantices (which are conventionally used for class names or instances)